### DISCUSSION TITLE
Searching Preset Feature

### DISCUSSION THREAD

**Discussion by trankhanh040147** [dailydotdev/daily#1096] [Feature Requests]
I hope that we can get a feature that allows us to store custom search/filter tags as presets. For example, I would like a preset for searching only "Golang + Interview"; on the other hand, I would like another preset for searching only "Docker + Kubernetes + CI/CD". Currently, the only possible way to do this is by changing settings in **Feed Settings** > **Manage Tags**, which is very inconvenient when I want to focus on specific tag groups. I hope you will take a moment to consider implementing this feature. Thank you. By the way, Daily.dev is so awesome!

**Comment by idoshamun** [comment]
Hi @trankhanh040147,

We have an active discussion about this feature here: https://github.com/dailydotdev/daily/discussions/1093
Thanks for the idea, let's keep the discussion there :)
